AK/Math.h

```diff
@@ -244,7 +248,9 @@ constexpr T sin(T angle)
         : "0"(angle));
     return ret;
 #else
-    return __builtin_sin(angle);
+    // FIXME: This is a very naive implementation, and is only valid for small x.
+    //        Probably a good idea to use a better algorithm in the future, such as a taylor approximation.
```

Comment:
This is a taylor approximation. "Such as a taylor approximation with more terms, and mapping large values by the function's period back to the origin" or somesuch.

(As-is is fine too, but slightly wrong.)